async port scaner.py

```diff
@@ -0,0 +1,31 @@
+import socket
+import asyncio
+import time
+t=time.time()
```

Comment:
Format file to be compatible with PEP8. Use Ctrl + Alt + L for that in PyCharm. Generally it will just reformat the spaces and tabs.